src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,58 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static final String DELIMITER = ",|:";
```

Comment:
```suggestion
    private static final String DELIMITER = ",|:";
```
상수활용 👍 

외부에 노출되지 않는 상수의 경우 private으로 접근을 막아주면 좋겠네요 :)